cmd/habitat/community/consensus/raft/dispatch.go

```diff
@@ -0,0 +1,13 @@
+package raft
+
+import "encoding/base64"
+
+type RaftDispatcher struct {
+	communityID    string
+	clusterService *ClusterService
+}
+
+func (d *RaftDispatcher) Dispatch(json []byte) error {
```

Comment:
same with this, why do we need the RaftDispatcher, do you think it will have more eventually or can we use the cluster service for this? i don't have a strong opinion, just curious.

Reply:
Main reason was so that I could write `LocalDispatcher` which lets us dispatch transitions immediately without it having to go through Raft, mainly for testing. It might also be useful if we set up a single node community mode.